worker/xml/xml_generator.py

```diff
@@ -0,0 +1,97 @@
+import logging
+import os
+import json
+
+from jinja2 import Template
+
+log = logging.getLogger(__name__)
+
+
+def generate_xml(work_path, data=None, xml_type='marc'):
```

Comment:
Default value immer als `None` und  dann über `if` defaulten. Siehe [hier](https://stackoverflow.com/questions/13195989/default-values-for-function-parameters-in-python)

Reply:
Ich würde mich für einen Fall entscheiden, entweder `data` muss immer übergeben werden oder gar nicht.